tests/e2e/tests/pilot/testdata/app.yaml.tmpl

```diff
@@ -18,28 +18,28 @@ spec:
   ports:
   - port: 80
     targetPort: {{.port1}}
-    name: http
+    name: foo
```

Comment:
do you want to name this as foo-http just for clarity and debugging ? and add a comment saying this should not be renamed to http-foo, as its used to test sniffing

Reply:
Comment added